proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -259,6 +265,22 @@ message QueryConsumerChainOptedInValidatorsResponse {
   repeated string validators_provider_addresses = 1;
 }
 
+message QueryConsumerChainConsumerValidatorsRequest {
+  string chain_id = 1;
+}
+
+message QueryConsumerChainConsumerValidator {
+  // The consensus address of the validator on the provider chain
+  string provider_address = 1 [ (gogoproto.moretags) = "yaml:\"address\"" ];
```

Comment:
```suggestion
  string provider_address = 1 [ (gogoproto.moretags) = 'yaml:"address"' ];
```

Reply:
for golang this is true but this is protobuf code. according to their lang spec for string literals it's equivalent.
(I gave it a try and the protobuf compiler produces the same output).
fine for me to keep the double quotes with escape chars - as we have it as such all over the place - as it was more of a nit comment.